quandl/get_table.py

```diff
@@ -37,4 +42,10 @@ def get_table(code, **options):
             break
         page_count = page_count + 1
         options['qopts.cursor_id'] = next_cursor_id
-    return data.to_pandas()
+
+    if returns == 'numpy':
+        return data.to_numpy()
```

Comment:
duplicate code - let's put this in a utility method so it can be consistent/shared with `quandl.get`?